app/models/user.rb

```diff
@@ -400,6 +400,10 @@ def has_devices?
     !recent_devices.empty?
   end
 
+  def new_device?(device_cookie:)
+    !device_cookie || !devices.exists?(cookie_uuid: device_cookie)
```

Comment:
In retrospect, `cookie_uuid` might have been a better argument name for me to have suggested, since it aligns with the table field and therefore might be more easily understood.

```suggestion
  def new_device?(cookie_uuid:)
    !device_cookie || !devices.exists?(cookie_uuid:)
```

Reply:
I might be missing another step but I get a missing keyword `':cookie_uuid'` error with the suggested code.

Reply:
You would need to also update all the call sites to pass `cookie_uuid:` as the argument instead of `device_cookie:`.

Reply:
Oh right, thank you!